ScifiCityscape-Robert

RobertoEchCCHS opened the pull request:
Write a short description about your art piece, such as what your artwork represents and your experience drawing it!

Next, make sure you go through **each item** in the following checklist. Delete this line after!

- [x] I have read the steps to [getting a blot](https://github.com/hackclub/blot/blob/main/docs/GET_A_BLOT.md)
- [x] I have read all the [submission requirements](https://blot.hackclub.com/submitting) and follow them 
- [x] I am submitting art that...
  - [x] is algorithmically generated (will meaningfully change each time the program is run)
  - [x] is your own work that is not copied from somewhere else
  - [x] is drawable through the editor(does not use outside tools)
  - [x] doesn't call `Math.random()` (See the documentation on [randomness](https://github.com/hackclub/blot/blob/main/public/TOOLKIT.md#randomness))
  - [x] is drawable on Blot itself (doesn't have lines overlap more than 5 times)
    - [x] Doesn't overlap lines more than 5 times
    - [x] Fits in the working area of 125x125mm
    - [x] Doesn't rely on colour to look good

The hardest part of this project was getting the line spacing just right to avoid too much overlap, while still keeping the design looking architectural. I learned how to balance the level of detail with the technical limits of the plotter by using things like different window patterns and careful spacing. This helped me add depth without making the plotter struggle.


Lastly, link a picture of your art piece here:
<img width="430" alt="0" src="https://github.com/user-attachments/assets/4dad750f-4778-4750-8a31-7fa4fed14db1">

RobertoEchCCHS commented:
Please review the more recent one (Nov 6), as the older one has errors!

RobertoEchCCHS commented:
I'll update this soon, gonna take a while, I was also wondering... how do I update the code? Create a new PR?? Ty!

EnterpriseGoose commented:
I look forward to seeing the updates! To update, just push the new code to
your fork and this pr will automatically update (including the preview
automation)

RobertoEchCCHS commented:
I just updated it, it took a while to figure out how to put the windows randomly and in the front! I also added some abstract clouds! Thank you so much for your patience, I hope to get a blot soon!

saumilthecode commented:
woah really cool drawing!

but some area for improvements could be that the bottom part and the sky looks really blank right now!

(i dont review this officialy but this doesn't look like it meets the weird 30h in to get blot criteria so you might want to fix this)

RobertoEchCCHS commented:
I just updated my stuff with the previous reviewer's fixes, please let me know if I can finally get a blot machine after 40 hours of work! Thank you!

saumilthecode commented:
![image](https://cloud-hnscao2gx-hack-club-bot.vercel.app/0image.png)
https://cloud-hnscao2gx-hack-club-bot.vercel.app/0image.png
(slack cdn cause the image is greater then 10mb somehow)

looks great though!
(for the "real" reviewer)

qcoral commented:
hi!! sorry for the long wait - this looks awesome and you can order a blot here:

https://airtable.com/appv0BzBY2APyIXj6/shrhpWVXN5imMzUmw
